### PR TITLE
feat: Add skill count and last skill add date for manager users

### PR DESCRIPTION
**Pull request by keabard**

### CODE REVIEW

File: back/Dockerfile [hunk 1]
```diff
@@ -5,7 +5,7 @@ MAINTAINER Jean-Louis Rigau <jlrigau@xebia.fr>
 RUN printf "deb http://archive.debian.org/debian/ jessie main\ndeb-src http://archive.debian.org/debian/ jessie main\ndeb http://security.debian.org jessie/updates main\ndeb-src http://security.debian.org jessie/updates main" > /etc/apt/sources.list
 
 RUN apt-get update
-RUN apt-get install bcrypt node-gyp -y
+RUN apt-get install bcrypt node-gyp -y --force-yes
```
Comment (blacroix) [inline back/Dockerfile:8]: I guess it's not needed? I want to know if there is an error ;)

File: web/src/components/Management/ManagerItem.js [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
+                      <span>Nombre de skills</span>
+                      <span>Dernier ajout de skill</span>
+                    </p>
                     {
                         manager.users.map((user, index) => (
-                            <p className="user" key={index} onClick={()=>onUserClick(user.readable_id)}>{user.name}</p>)
+                            <div className="manager_table_row" key={index} onClick={()=>onUserClick(user.readable_id)}>
+                                <span>{user.name}</span>
+                                <span>{user.skill_count}</span>
+                                <span>{moment(user.last_skill_update).format('YYYY-MM-DD')}</span>
```
Comment (blacroix) [inline web/src/components/Management/ManagerItem.js:27]: Why not 🇫🇷  format? `DD/MMM/YYYY`?

**Review by blacroix — CHANGES_REQUESTED** [review]

File: back/Dockerfile (hunk 1 above)
Comment (keabard) [reply back/Dockerfile:8]: There is an error when you don't put --force-yes :(

File: web/src/components/Management/ManagerItem.js (hunk 2 above)
Comment (keabard) [reply web/src/components/Management/ManagerItem.js:27]: I've used the format already present in the application.

Would you like me to change the date format everywhere else as well ?

File: back/Dockerfile (hunk 1 above)
Comment (blacroix) [reply back/Dockerfile:8]: Is it working even with the error? I.e. app is running?

File: web/src/components/Management/ManagerItem.js (hunk 2 above)
Comment (blacroix) [reply web/src/components/Management/ManagerItem.js:27]: As you want :)

**Review by blacroix — APPROVED** [review]

**Pull request merged**